hadoop-ozone/recon/src/main/java/org/apache/hadoop/ozone/recon/fsck/ContainerHealthTask.java

```diff
@@ -340,6 +340,12 @@ private void processContainer(ContainerInfo container, long currentTime,
       if (h.isMissing() && containerDeletedInSCM(container)) {
         return;
       }
+
+      if (h.getContainer().getState() == HddsProtos.LifeCycleState.DELETING) {
```

Comment:
Do we need this extra if condition check ? Can we just modify above if condition "if containerDeletedInSCM", then just return ? Or Am I missing something here ?